core/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -384,6 +384,18 @@ private void declareParseTemplate(ExtensionDeclarer extensionDeclarer, ClassType
         .ofType(typeLoader.load(String.class))
         .withExpressionSupport(NOT_SUPPORTED)
         .describedAs("The location of the template. The order in which the processor will attempt to load the file is: from the file system, from a URL, then from the classpath.");
+
```

Comment:
Now if the operation can return parsed templates in their correct mimeType, those parsed templates won't be just simple Strings, these could be Objects, Arrays, Anythings, so the new Output of the operation should be `Any` instead of `String` -> https://github.com/mulesoft/mule/blob/bb8e65fac6bdafc9f27a76a51affcea66b4d95ad/core/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java#L372

Reply:
Wouldn't they be Strings in different formats?

Reply:
I would need to check again but as i understood it, the parseLogTemplate() method always returns a String.